src/Dfc.CourseDirectory.FindACourseApi/FeatureFlags.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Dfc.CourseDirectory.FindACourseApi
+{
+    public static class FeatureFlags
```

Comment:
For now at least they are sourced from two different config files so can be different. I'd like to centralise them in one place but didn't want to do it in this PR.

Reply:
> For now at least they are sourced from two different config files so can be different.

But in reality they're reading the same values from our CI pipeline.

> I'd like to centralise them in one place but didn't want to do it in this PR.

You've moved all of the other `FeatureFlags` code in this PR - I think this is the right time to do it instead of duplicating it and trying to find time to come back to fix it later.